src/Thumbnail.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classSet from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+
+const Thumbnail = React.createClass({
+	mixins: [BootstrapMixin],
+
+	getDefaultProps() {
+		return {
+			bsClass: 'thumbnail'
+		};
+	},
+
+	render() {
+		let classes = this.getBsClassSet();
+
+		return (
+			<a {...this.props} className={classSet(this.props.className, classes)}>
```

Comment:
I'm thinking that if there is an `href` prop than use an anchor, otherwise use a div. Also, have you considered having `src` and `alt` props and creating the img tag within this component?

So the external api would be something like:

```
<Thumbnail src='...' alt='...' href='...' />

// Which produces:

<a href="..." class="thumbnail">
  <img src="..." alt="...">
</a>
```

OR

```
<Thumbnail src='...' alt='...'>
  <h3>Thumbnail label</h3>
  <p>...</p>
  <p>
    <a href="#" class="btn btn-primary" role="button">Button</a>
    <a href="#" class="btn btn-default" role="button">Button</a>
  </p>
</Thumbnail>

// Which Produces: (Note the caption div)

<div class="thumbnail">
  <img src="..." alt="...">
  <div class="caption">
    <h3>Thumbnail label</h3>
    <p>...</p>
    <p><a href="#" class="btn btn-primary" role="button">Button</a> <a href="#" class="btn btn-default" role="button">Button</a></p>
  </div>
</div>
```